### PR TITLE
remove eclipse from docs

### PR DESCRIPTION
**Pull request by KirbyKatcher**

### CODE REVIEW

**Comment by galasa-team** [comment]
Build successful

File: README.md [hunk 1]
```diff
@@ -399,12 +383,12 @@ pr-obr-generic:
 The pipeline first verifies that the author of the PR to the repository is either an approved code-committer or code-admin, before attempting to build any code. It then clones the Automation, Framework, Extensions and Managers repositories at the main branch, and the OBR repository at the PR's branch. galasabld generates a pom for OBR generic and it is built with Maven, then a Docker image built. Property files are then copied to build the the runtimes of Galasa. The status of the pipeline is then returned to the PR. 
 
 branch-obr-generic:
-This pipeline follows the same steps as the pr-obr-generic pipeline other than verifying the author of or returning status back to a PR. It then triggers either the branch-eclipse pipeline or the branch-isolated pipeline depending on whether a Main build or Complete build is occurring.
+This pipeline follows the same steps as the pr-obr-generic pipeline other than verifying the author of or returning status back to a PR. It then triggers either the branch-isolated pipeline.
```
Comment (jadecarino) [inline README.md:386]: My bad, this README.md is a little out of date so this actually should say that it then triggers the branch-helm and run-tests pipelines if doing a Main build, or branch-cli if doing a Complete build. Can you update please?

**Review by jadecarino — CHANGES_REQUESTED** [review]
One small change but just because the doc was a little out of date anyway. I'll review the whole README.md properly soon!

**Review by jadecarino — APPROVED** [review]

**Comment by galasa-team** [comment]
Build successful

**Pull request merged**